content/inside-rust/call-for-testing-hint-mostly-unused.md

```diff
@@ -0,0 +1,194 @@
++++
+path = "inside-rust/2025/07/14/call-for-testing-hint-mostly-unused"
+title = "Call for Testing: Speeding up compilation with `hint-mostly-unused`"
+authors = ["Josh Triplett"]
++++
+
+I'm pleased to announce, and call for testing of, the nightly-only `rustc`
+`-Zhint-mostly-unused` option, and the corresponding nightly Cargo features
+`profile.hint-mostly-unused` and `hints.mostly-unused`. These options can help
+accelerate your Rust compile time in some cases, by avoiding compilation of
+items from your dependencies that you aren't using. Your feedback will help
+evaluate these features and make progress towards stabilizing them in the
+future.
+
+## Background
+
+Some crates provide comprehensive APIs with a very large surface area, yet many
```

Comment:
Perhaps it will help intuition to say this is for APIs that are monomorphic? Not sure how to put that.

Reply:
I attempted to cover this; let me know how the new version looks.